egs/librispeech/asr/simple_v1/mmi_att_transformer_decode.py

```diff
@@ -34,63 +41,95 @@
 from snowfall.training.mmi_graph import get_phone_symbols
 
 
+def decode_one_batch(batch: Dict[str, Any],
+                     model: AcousticModel,
+                     HLG: k2.Fsa,
+                     output_beam_size: float,
+                     num_paths: int,
+                     use_whole_lattice: bool,
+                     G: Optional[k2.Fsa] = None)->Dict[str, List[List[int]]]:
```

Comment:
should probably document this if we intend to check it in.

Reply:
I've added some documentation to it.